app/assets/stylesheets/devise/registrations/edit.css

```diff
@@ -0,0 +1,24 @@
+.align-self-center {
+	text-align: center;
+}
+
+#edit-user-container {
+	margin: 0px;
```

Comment:
This may sound a little unreasonable, but let's try to write our css rules alphabetically. I don't mean the whole document must be alphabetic, just inside each class or id, write the attributes alphabetically, please. This is going to save us a lot of time later when debugging the styles.

Also, and I'm quoting just about every web developer I've known here, "Never use Ids for styling, use classes".